lib/omnibus/packagers/rpm.rb

```diff
@@ -830,5 +832,25 @@ def remove(packages)
         shellout!("yum -y remove #{packages}")
       end
     end
+
+    def compression_level(val = nil)
+      unless val.nil?
+        unless val >= 0 && val <= 9
+          raise InvalidValue.new(:compression_level, 'be an Integer between 0 and 9 included')
+        end
+      end
+      @compression_level = val || 7
```

Comment:
if 7 is the default value:
* is there any reason why the values differ for `deb` and `rpm` (methods could be on a common location to prevent code duplication, wdyt)?
* why don't put 9 as default value?

Reply:
Regarding the defaults, for debian it comes from `dpkg-deb`'s default value:
```
       -zcompress-level
           Specify which compression level to use on the compressor
           backend, when building a package (default is 9 for gzip, 6
           for xz, 3 for zstd).  The accepted values are compressor
           specific.  For gzip, from 0-9 with 0 being mapped to
           compressor none.  For xz from 0-9.  For zstd from 0-22, with
           levels from 20 to 22 enabling its ultra mode.  Before dpkg
           1.16.2 level 0 was equivalent to compressor none for all
           compressors.
```

`rpmbuild` on the other hand relies on the default value from the `xz` tool, which appears to be `6` or `7` depending on the docs (I'm just realizing this now, so I'll update to 6 everywhere)

9 as a default value is probably too agressive for a rather minimal gain. It would basically double the compression time to save a few MB, which is not worth the CPU cost in regular PR pipelines. It's worth it for releases as every byte is expensive there (see the charts in https://docs.google.com/document/d/1OPATMVtLjny9IW68Q27_0QlxlsgfhOSeQoSi3gQJOwI/edit?pli=1#heading=h.1pff0synl7yk)

Finally as far as code duplication is concerned, I agree, but would rather do this at a later point once the change is accepted and merged